lib/src/widgets/phone_field.dart

```diff
@@ -113,6 +113,7 @@ class PhoneField extends StatefulWidget {
     required this.restorationId,
     required this.enableIMEPersonalizedLearning,
     required this.inputFormatters,
+    required this.showDialCode,
```

Comment:
Since this is part of the public API a test should be added

Reply:
What do you mean?

Reply:
Could you add a test in the test file that verify that when show dial code is true, the dial code is shown, and when it is false it is not ?

Reply:
ok i do it